projectfiles/CodeBlocks/README.md

```diff
@@ -9,16 +9,16 @@
 2.  Install CodeBlocks from <http://www.codeblocks.org/>.
     MinGW is not needed.
 
-3.  Install tdm-gcc-5.1.0 from <http://sourceforge.net/projects/tdm-gcc/files/TDM-GCC%20Installer/>.
+3.  Download and unpack i686-7.2.0-release-posix-dwarf-rt_v5-rev0.7Z or i686-7.2.0-release-posix-sjlj-rt_v5-rev0.7Z
```

Comment:
Instead of file names, I'd prefer direct download links:

https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/7.2.0/threads-posix/dwarf/i686-7.2.0-release-posix-dwarf-rt_v5-rev0.7z
https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/7.2.0/threads-posix/sjlj/i686-7.2.0-release-posix-sjlj-rt_v5-rev0.7z